### PR TITLE
fix empty settings after importing backup

### LINKED ISSUE #2260: Empty "Settings" window displayed
- Operating System (Linux/Mac/Windows/iOS/Android):
Windows 8.1 Pro.

- Delta Chat Version:
1.20.0.

- Expected behavior:
The "Settings" window displays control elements and input fields.

- Actual behavior:
The "Settings" window is empty.

- Steps to reproduce the problem:
-- Run Delta Chat from scratch.
-- Import a(n Android) backup.
-- Open "Settings" from the three-dot menu.

- Screenshots:
grafik

- Logs:
N/A.

### PR DESCRIPTION
**Pull request by Simon-Laux**
fix #2260

### CODE REVIEW

**Comment by gerryfrancis** [comment]
@Simon-Laux Sorry, I am no coder, please let someone else review your PR. ;) But many thanks for the fix! :+1:

File: src/renderer/components/LoginScreen.tsx [hunk 1]
```diff
@@ -58,9 +57,7 @@ function ImportBackupProgressDialog({
         return
       }
       onClose()
-      if ((await DeltaBackend.call('login.loadAccount', account)) === true) {
-        window.__changeScreen(Screens.Main)
-      }
+      window.__loadAccount(account)
```
Comment (Jikstra) [inline src/renderer/components/LoginScreen.tsx:60]: Can you describe what the problem was? I guess this is the bug that happend with the iphone backup?

Comment (gerryfrancis) [reply src/renderer/components/LoginScreen.tsx:60]: > I guess this is the bug that happend with the iphone backup?

@Jikstra No, the backup was done on Android.

Comment (Simon-Laux) [reply src/renderer/components/LoginScreen.tsx:60]: The change screen function skipped loading in the account variable so it was missing so the settings did not load for any session that was started by an backup import, sometimes there was even information of the last loaded account in the settings dialog. this pr changes the logic to use the right function for the task.

**Comment by Simon-Laux** [comment]
> @Simon-Laux Sorry, I am no coder, please let someone else review your PR. ;) But many thanks for the fix! +1

I the check there are prebuilds that can be downloaded and tested if you want ;)

**Comment by gerryfrancis** [comment]
@Simon-Laux Thank you so much, I checked out the preview, the "Settings" window loads/displays fine now! :+1:

**Pull request merged**